test/service/test_manifest.py

```diff
@@ -433,6 +433,17 @@ def test_manifest_zarr(self):
                     # 2 pairs of files out of the 13 have matching checksums
                     self.assertEqual({f['sha256'] for f in expected}, {row['file_sha256'] for row in rows})
                     self.assertEqual({f['crc32c'] for f in expected}, {row['file_crc32c'] for row in rows})
+                    # Manifest for `curl` format  should also contain related files
```

Comment:
I personally don't mind it, but the little Hannes in my head is saying that this comment doesn't add anything that can't already be deduced from the code. Consider deleting it.